src/main/java/seedu/socket/logic/parser/ParserUtil.java

```diff
@@ -246,4 +246,16 @@ public static ProjectMeeting parseMeeting(String meeting) throws ParseException
         }
         return new ProjectMeeting(trimmedName);
     }
+
+    /**
+     * Parses {@code Collection<String> meetings} into a {@code Set<ProjectMeeting>}.
+     */
+    public static Set<ProjectMeeting> parseMeetings(Collection<String> meetings) throws ParseException {
```

Comment:
Out of curiosity, is it necessary for `parseMeetings` to return a set in the current implementation?

Also, perhaps want to consider including a test for this method under `ParserUtilTest.java`

Reply:
There isn't a need to currently, the `parseMeeting` method in `ParserUtil.java` handles the parsing of user's input. I have added it here for possible future extensions.

After considering, maybe it would be better if I remove it, since I don't think it is wise to create methods that isn't use, to prevent possible error usage or abuse.

Will update it when I'm free.